src/codegen/codegen_cuda.cc

```diff
@@ -50,6 +50,9 @@ void CodeGenCUDA::AddFunction(LoweredFunc f) {
 
 std::string CodeGenCUDA::Finish() {
   if (enable_fp16_) {
+    static constexpr const char* _cuda_half_t_def =
+    #include "literal/cuda_half_t.txt"
```

Comment:
consider still keep it as a header file so we can apply the license header, define a global constant

Reply:
@tqchen addressed.